beetmoverscript/templates/thunderbird_candidates_repacks.yml

```diff
@@ -18,6 +18,10 @@ mapping:
       s3_key: {{ platform }}/xpi/{{ locale }}.xpi
       destinations:
         - {{ platform }}/xpi/{{ locale }}.xpi
+    buildhub.json:
+      s3_key: {{ platform }}/{{ locale }}.buildhub.json
+      destinations:
+        - {{ platform }}/{{ locale }}.buildhub.json
```

Comment:
I assume you meant `{{ platform }}/{{ locale }}.buildhub.json` here instead in both occurrences.

Reply:
I believe I have that?